src/jit/compiler.h

```diff
@@ -1933,9 +1933,10 @@ class Compiler
     GenTreePtr gtNewIndexRef(var_types typ, GenTreePtr arrayOp, GenTreePtr indexOp);
 
     GenTreeArgList* gtNewArgList(GenTreePtr op);
-
     GenTreeArgList* gtNewArgList(GenTreePtr op1, GenTreePtr op2);
 
+    GenTreeArgList* gtNewAggregate(GenTree* element);
```

Comment:
Ping--aside from nomenclature, this change is ready. If we can agree on a name rather quickly I'm happy to block on that, but otherwise I'd like to get this checked in and follow up on the naming later.